### ISSUE TITLE
Testing SyntaxNet model trained for UD_Spanish-AnCora, not found char-ngram-map & morphology-map.

### ISSUE THREAD

**Issue by TheRealTau** [tensorflow/models#538]
Training my own SyntaxNet model for Spanish-Ancora UD corpus following the instructions from [here](https://github.com/tensorflow/models/tree/master/syntaxnet) did not give me errors.. When I try to test it using `parser.sh` from syntaxnet/models/parsey_universal dir the result was the missing files error.

```
F syntaxnet/term_frequency_map.cc:63] Check failed: ::tensorflow::Status::OK() == (tensorflow::Env::Default()->NewRandomAccessFile(filename, &file)) (OK vs. Not found: syntaxnet/models/parsey_universal/modeltest/char-ngram-map)
F syntaxnet/term_frequency_map.cc:63] Check failed: ::tensorflow::Status::OK() == (tensorflow::Env::Default()->NewRandomAccessFile(filename, &file)) (OK vs. Not found: syntaxnet/models/parsey_universal/modeltest/morphology-map) 
```

So i check the files produced in the training phase and the final files it creates were:

```
-category-map  
-char-map  
-checkpoint  
-context  
-graph  
-label-map  
-latest-model  
-latest-model.meta  
-lcword-map  
-model  
-model.meta  
-prefix-table  
-status  
-suffix-table  
-tag-map  
-tag-to-category  
-tagged-dev-corpus  
-tagged-training-corpus  
-tagged-tunning-corpus  
-word-map 
```

The context.pbtxt file used for the training was the one from syntaxnet/models/parsey_universal.

Then I download the Spanish pretrained model from[ here](https://github.com/tensorflow/models/blob/master/syntaxnet/universal.md). And I checked the files. It seems like there are two missing files, the pretrained model has it by default but in the one I trained, these files were missing.

So my questions are how can I get this files in the SyntaxNet training phase?
There are other ways to produce them?
How can I test my SyntaxNet model trained?

**Comment by drpngx** [comment]
@calberti is there a problem with the data pack?

**Comment by jimmy1357** [comment]
I met the same error when I was training a Chinese model by UD_Chinese corpus.
The error message was:
`F syntaxnet/term_frequency_map.cc:63] Check failed: ::tensorflow::Status::OK() == (tensorflow::Env::Default()->NewRandomAccessFile(filename, &file)) (OK vs. Not found: /root/jichen/models/syntaxnet/models/brain_parser/structured/200x200-0.02-100-0.9-0/label-map)
`
I copied the label-map from the brain_pos model , then run the test shell again, another error appeared,
`F syntaxnet/term_frequency_map.cc:63] Check failed: ::tensorflow::Status::OK() == (tensorflow::Env::Default()->NewRandomAccessFile(filename, &file)) (OK vs. Not found: /root/jichen/models/syntaxnet/models/brain_parser/structured/200x200-0.02-100-0.9-0/char-ngram-map)
F syntaxnet/term_frequency_map.cc:63] Check failed: ::tensorflow::Status::OK() == (tensorflow::Env::Default()->NewRandomAccessFile(filename, &file)) (OK vs. Not found: /root/jichen/models/syntaxnet/models/brain_parser/structured/200x200-0.02-100-0.9-0/morphology-map)`

So i check the files produced in the training phase and the final files it creates were:
`beam-parsed-dev-corpus       beam-parsed-tuning-corpus  context  latest-model       model       status
beam-parsed-training-corpus  checkpoint                 graph    latest-model.meta  model.meta`

The files of the trained model from "http://download.tensorflow.org/models/parsey_universal/Chinese.zip" were:

`char-map        checkpoint  last-word-map    morpher-params  parser-params  suffix-table  tagger-params     tokenizer-params.meta
char-ngram-map  label-map   morph-label-set  morphology-map  prefix-table   tag-map       tokenizer-params  word-map` 

How to solve the problem?
Thank you very much!

**Comment by jqqcq** [comment]
My situation is the same as yours. @jimmy1357 ，Have you found a way to solve it now 
Thank you very much!

**Comment by adollo** [comment]
My situation is the same as yours. @jimmy1357 ，Have you found a way to solve it now
Thank you very much!

**Comment by adollo** [comment]
I don't know how to train to product tokenizer-params(models). Do you have any idea?@jimmy1357

**Comment by inferrna** [comment]
@xtknight suggested an working solution for producing both morph-label-set and morphology-map, see https://github.com/tensorflow/models/issues/830

**Comment by mingxingtan** [comment]
This is a duplication of #830.
